tests/src/Unit/OgAccessTest.php

```diff
@@ -21,17 +21,22 @@ class OgAccessTest extends OgAccessTestBase {
    */
   public function testUserAccessNotAGroup($operation) {
     $this->groupManager->isGroup($this->entityTypeId, $this->bundle)->willReturn(FALSE);
-    $user_access = OgAccess::userAccess($this->groupEntity()->reveal(), $operation);
+    $user_access = OgAccess::userAccess($this->group, $operation);
     $this->assertTrue($user_access->isNeutral());
   }
 
   /**
    * @coversDefaultmethod ::userAccess
    * @dataProvider operationProvider
    */
-  public function testUserAccessForbiddenByDefault($operation) {
-    $user_access = OgAccess::userAccess($this->groupEntity()->reveal(), $operation, $this->user->reveal());
-    $this->assertTrue($user_access->isForbidden());
+  public function testAccessByOperation($operation) {
+    $user_access = OgAccess::userAccess($this->group, $operation, $this->user->reveal());
+
+    // We populate the allowed permissions cache in
+    // OgAccessEntityTestBase::setup().
+    $condition = $operation == 'update group' ? $user_access->isAllowed() : $user_access->isForbidden();
```

Comment:
It's actually being populated in the parent class `OgAccessTestBase::setup()` instead of in `OgAccessEntityTestBase::setup()`. I guess it is technically correct, but I went looking for the code and didn't find it there initially :)